PLAYER/RestorePlayerStamina.md

```diff
@@ -7,9 +7,21 @@ ns: PLAYER
 // 0xA352C1B864CAFD33 0x62A93608
 void RESTORE_PLAYER_STAMINA(Player player, float p1);
 ```
+Regenerate a players stamina. Unsure if p1 is some sort of multiplier or regen speed, need to look into this 🤔.
```

Comment:
I just ran some tests and based on my research, `p1` seems to be a percentage that ranges from 0.0 to 1.0 (1.0 being 100%) and it seems to add a percentage of stamina to the player.

If the player's stamina is 10.0 and you restore 10% stamina (0.10) you get 20% stamina on the player.

This is the code I used to run my tests:
```lua
-- Inside a while loop
Citizen.Trace("Stamina: " .. GetPlayerStamina(PlayerId()))

-- Command
RegisterCommand("rs", function(source, args, rawCommand)
  RestorePlayerStamina(PlayerId(), 0.1)
end, false)
```

The game's native method (my name guesses may be wrong):

```c
void __fastcall restoreStamina(__int64 ped, float percentageToRestore)
{
  float currentStaminaValue; // xmm2_4
  float newStaminaValue; // xmm0_4

  if ( percentageToRestore >= 0.0 && percentageToRestore <= 100.0 )// This is wrong, should be <= 1.0
  {
    currentStaminaValue = *(float *)(ped + 0xBB4);
    newStaminaValue = (float)(currentStaminaValue * percentageToRestore) + *(float *)(ped + 0xBB0);
    *(float *)(ped + 0xBB0) = newStaminaValue;
    if ( newStaminaValue >= currentStaminaValue )
      *(float *)(ped + 2992) = currentStaminaValue;
  }
}
```

Reply:
I'll update the documentation 👍🏼